### ISSUE TITLE
Wrong project location when opening workspace

### ISSUE THREAD

**Issue by leoravagnani** [CocoaPods/CocoaPods#5160]
I haven't filled the template form since it's not a bug or recurring issue.

I'm working on a project where I integrated Cocoa Pods a couple months after I first started it and then Git versioning a couple months after that. On the workstation it was created it's all good, but if I check out the repo in another one, upon opening my workspace the Pods project is ok but my project is missing (it's icon is faded in the navigator). 

The file inspector has the answer: the project full path is pointing to a folder that only exists on the workstation it was created.

I tried setting it just as the Pods project is set, in a relative way, but got no luck, sometimes Xcode just turns my project into a folder structure, or kinda resolve it but open another project window and ignores the workspace altogether, and one single time I happened to close both the workspace and this separate project window, and upon reopening the workspace all was good, but I wasn't under version control and it was way outdated (had copied the project from a pen drive some weeks earlier - dumb, I know).

Now I'm not sure if it was a CocoaPods thing when wrapping the project on a workspace, or if it's the way it was set on the original project. I tried several times to adjust it on the original workstation, but again no luck.

I'm bummed I can't get to work when not sitting on a specific computer, it defeats one of the main purposes of versioning, but I feel I'm missing something and maybe there's an easy fix... any ideas?

Thanks in advance!

**Comment by segiddins** [comment]
You'll need to run `pod install` on the other machine if you're not checking in the `Pods` directory. In future, questions like this ought to be directed to stack overflow, since it's not about a bug in CocoaPods itself.

**Comment by leoravagnani** [comment]
The Pods directory is under versioning and it's project within the workspace is set just fine on any machine. The problem lies on my project, it's path is not set right, and I can't figure how to configure it to a relative path.

I was reading through the FAQs after submitting the issue and figured this should be on SO, I'll start a thread there, sorry for that!

(Any chance this could be something with how the project was wired when CocoaPods created the workspace? Hence maybe being an issue with my version of CocoaPods when setting up this workspace.)

**Comment by segiddins** [comment]
What happens if you `pod deintegrate && pod install`?

**Comment by leoravagnani** [comment]
Should I run this on the original machine or where I clone the project?

**Comment by orta** [comment]
Either. They both should have enough information to do it.

**Comment by leoravagnani** [comment]
Runnings the commands didn't do it for me.

I cloned the project to another folder on the original machine and as it "should" the project reference was set wrong, so I started fiddling with it and after quite some tries I somehow got it right (and I can't tell exactly what was it haha). I set up a new branch with the fix and cloned from it in several different locations on my machine and it's perfect now. I'll merge with my master branch and sync all others branches.

Now I don't know if it was related to CocoaPods but it's fixed!

Thanks to both of you guys for your time! And next time I'll post on SO ;)
